README.md

```diff
@@ -298,6 +298,41 @@ Content template of $system_auth_ac_file. If undef, parameter is set based on th
 
 - *Default*: undef, default is set based on OS version
 
+password_auth_file
+----------------
+Path to password-auth. Used on RedHat.
+
+- *Default*: '/etc/pam.d/password-auth'
+
+password_auth_ac_file
+-------------------
+Path to password-auth-ac. Used on RedHat.
+
+- *Default*: '/etc/pam.d/password-auth-ac'
+
+pam_password_auth_lines
+-------------------------
+Content template of $password_auth_ac_file. If undef, parameter is set based on the OS version.
```

Comment:
These should say something like the following, so that people know what type the parameter should be.

`Array of lines used in content template of $password_auth_ac_file. If undef, parameter is set based on defaults for the detected platform.`